src/commands/bootcamp/bootcampNextMentee.ts

```diff
@@ -0,0 +1,79 @@
+import { Message, VoiceChannel, TextChannel } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { MentorCommand } from '../../utils/commands';
+import { BootcampSettings } from '../../components/bootcamp';
+
+class BootcampNextCommand extends MentorCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'next',
+      aliases: ['mentee'],
+      group: 'bootcamp',
+      memberName: 'next',
+      guildOnly: true,
+      description: 'Moves the next mentee in line to your call.'
+    });
+  }
+
+  async onRun(message: CommandoMessage): Promise<Message> {
+    const mentor = message.member?.voice;
+    const callChannel = mentor?.channel;
+    const guild = message.guild;
+    const track = callChannel?.parent?.name;
+
+    let front;
+
+    const queueVoice = <VoiceChannel>(
+      guild.channels.cache.find((channel) => channel.name === track && channel.type === 'voice')
+    );
+    const queueChannel = <TextChannel>(
+      callChannel?.parent?.children.find(
+        (channel) => channel.name === track?.replace(/ +/g, '-').toLocaleLowerCase() + '-queue'
+      )
+    );
+    const queueMembers = queueVoice?.members;
+
+    if (queueVoice && queueChannel) {
+      if (callChannel?.full) {
+        return message.say('You should only have one Mentee at a time.');
+      }
+      if (queueMembers.size < 1) {
+        return message.say('The are no Mentees waiting for this track.');
+      }
+
+      queueChannel.messages.fetch({ limit: queueVoice.members.size * 2 + 5 }).then((messages) => {
+        messages
+          .sorted((mesgA, mesgB) => mesgA.createdTimestamp - mesgB.createdTimestamp)
```

Comment:
I can see this running into performance issues with large enough numbers, but a fix would be non trivial given our timelines